### PR TITLE
Tweaking styles

### PR DESCRIPTION
**Pull request by shazow**
Also added a nix shell config, hope you don't mind. :)

Screenshot:
image

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -7,14 +7,20 @@ import { createGlobalStyle } from 'styled-components'
...
+	background: rgba(51,51,51,1);
 }
 
-body {
-	background: rgba(51,51,51,1);
+a {
+    color: rgba(248,255,166,0.8);
+}
+
+#__next {
```
Comment (shazow) [inline pages/_app.tsx:21]: Not sure if this is the correctest way to do this, but seems like this is what people do.

**Comment by shazow** [comment]
Made the buttons more juicy

image

**Comment by shazow** [comment]
Rebased, also more subtle separators

image

**Comment by shazow** [comment]
@dt665m Feel free to merge/deploy once you get a chance to review please. :)

**Review by dt665m — APPROVED** [review]
lgtm gonna merge quickly so I can also merge the font branch

**Pull request merged**